server.js

```diff
@@ -573,7 +573,11 @@ cache(function (data, match, sendBadge, request) {
 
   fetchFromSvg(request, url, function (err, res) {
     if (err != null) {
-      badgeData.text[1] = defaultOpts.inaccessibleLabel;
+      if (err.message.includes("Can't get value from SVG")) {
+        badgeData.text[1] = 'invalid';
```

Comment:
Huh, when would this happen? Is this the 404 case?

Do you think the erroring would be clearer if we fetch the SVG source from within the badge, and invoke `valueFromSvgBadge` only upon success?